ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/NamespaceView/NamespaceViewPage.tsx

```diff
@@ -37,7 +161,149 @@ function NamespaceViewPage() {
                 </Flex>
             </PageSection>
             <Divider component="div" />
-            <PageSection padding={{ default: 'noPadding' }}></PageSection>
+            <PageSection>
+                <Toolbar>
+                    <ToolbarContent>
+                        <FilterAutocompleteSelect
+                            searchFilter={searchFilter}
+                            onFilterChange={(newFilter) => setSearchFilter(newFilter)}
+                            searchOptions={searchOptions}
+                        />
+                        <ToolbarItem variant="pagination" alignment={{ default: 'alignRight' }}>
+                            <Pagination
+                                toggleTemplate={({ firstIndex, lastIndex }) => (
+                                    <span>
+                                        <b>
+                                            {firstIndex} - {lastIndex}
+                                        </b>{' '}
+                                        of <b>many</b>
+                                    </span>
+                                )}
+                                page={page}
+                                perPage={perPage}
+                                onSetPage={(_, newPage) => setPage(newPage)}
+                                onPerPageSelect={(_, newPerPage) => setPerPage(newPerPage)}
+                                isCompact
+                            />
+                        </ToolbarItem>
+                        <ToolbarGroup aria-label="applied search filters" className="pf-u-w-100">
+                            <SearchFilterChips
+                                onFilterChange={onFilterChange}
+                                filterChipGroupDescriptors={searchOptions.map(
+                                    ({ label, value }) => {
+                                        return {
+                                            displayName: label,
+                                            searchFilterName: value,
+                                        };
+                                    }
+                                )}
+                            />
+                        </ToolbarGroup>
+                    </ToolbarContent>
+                </Toolbar>
+                <TableComposable borders={false}>
+                    <Thead noWrap>
+                        <Tr>
+                            <Th sort={getSortParams('Namespace')} width={30}>
+                                Namespace
+                            </Th>
+                            <Th sort={getSortParams('Namespace Risk Priority')}>Risk priority</Th>
+                            <Th sort={getSortParams('Cluster')}>Cluster</Th>
+                            <Th sort={getSortParams('Deployment Count')}>Deployments</Th>
```

Comment:
Does `Deployment Count` work as a sort field for namespaces? I wasn't able to get it to work correctly for "Images" on the main list page.

Reply:
When I tested it out, it was working. I noticed issues with the sort when planning out what the query should look like for this component. Charmik made a fix on the backend some time ago. That might have fixed it for you too.